.github/workflows/release.yaml

```diff
@@ -369,3 +369,36 @@ jobs:
 
       - name: Update latest tag
         uses: EndBug/latest-tag@latest
+
+  release_homebrew:
+    needs: release_gh
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get release version and hashes
+        run: |
+          echo "::set-output name=version::$(grep -m 1 version Cargo.toml | cut -d' ' -f3 | tr -d '\"' | cut -d'-' -f1)"
```

Comment:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Now seems like a good time to migrate to the env vars, before set-output becomes more prevalent